templates/webapps/tool_shed/repository/common.mako

```diff
@@ -1071,6 +1071,40 @@
     %>
 </%def>
 
+<%def name="render_module_resolver( resolver_dependencies )">
```

Comment:
Can you reword some of these variables and text, my initial thoughts are as follows:

- ``render_module_resolver`` -> ``render_tool_dependency_resolver``
- ``<b>Model Class</b>`` -> - ``<b>Dependency Resolver Type</b>``
- ``Module Resolver Details`` -> - ``<b>Dependency Resolver Details</b>``